src/structure.rs

```diff
@@ -137,11 +140,23 @@ fn check_package(
     } else {
         let package_name_valid = PACKAGE_NAME_REGEX.is_match(&package_name);
         let result = if !package_name_valid {
-            npv_141::InvalidPackageDirectoryName::new(
-                package_name.clone(),
-                relative_package_dir.clone(),
-            )
-            .into()
+            if package_name
```

Comment:
Think this if-expression can be `if package_name.starts_with(|c: char| c.is_ascii_digit())`